cs251tk/toolkit/args.py

```diff
@@ -47,12 +53,11 @@ def get_args():
     folder.add_argument('--no-update', '-n', action='store_true',
                         help='Do not update the student folders when checking')
     folder.add_argument('--stogit', metavar='URL',
-                        default='git@stogit.cs.stolaf.edu:sd-s17',
-                        help='Use an alternate stogit base URL')
+                        help='Use an alternate stogit base URL (eg, git@stogit.cs.stolaf.edu:sd-s17)')
```

Comment:
e.g. is twice as long on a terminal, so I usually abbreviate to eg purely for space.